ops/model.py

```diff
@@ -766,7 +767,12 @@ class Port:
     """The port number. Will be ``None`` if protocol is ``'icmp'``."""
 
 
-OpenedPort = Port  # Alias for backwards compatibility.
+class OpenedPort(Port):
```

Comment:
Per discussion on Mattermost, I think we should revert this: I don't think it's a good precedent to break backwards compatibility (in the case of `isinstance`) just for a minor docstring improvement. (Though I realize it's unlikely anyone's actually doing that with this particular class.)